docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -2482,4 +2482,39 @@ None
         "id": 1,
         "result": "Success"
     }
-    ```
\ No newline at end of file
+    ```
+    
+## Miscellaneous Methods 
+
+### rpc_modules
```

Comment:
Perhaps explain the use of this method a bit more like it's used by Geth Console to know what it can call. SOme other projects have the same question about what it's used for : https://github.com/ethereum/web3.py/issues/254

Reply:
Per Danno the fact this is used by geth console has no bearing on it's implementation so he thought we should leave that out.  It's unlikely to be called by anyone but after discussion with the POs I'm going to leave it in the doc for completeness.